scripts/utils/splitLandIdsForTunnelMigration.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs-extra';
+
+const tokensSnapshotL1 = JSON.parse(
+  fs.readFileSync('./tunnel_mainnet.json').toString()
+);
+const tokensSnapshotL2 = JSON.parse(
+  fs.readFileSync('./tunnel-polygon.json').toString()
```

Comment:
```./tunnel-polygon.json``` => ```./tunnel_polygon.json```

Reply:
Using the name of the files which Leon gave.